assessments/guide/joint-readme-template.md

```diff
@@ -0,0 +1,46 @@
+# [Project Name] Security Evaluation
```

Comment:
I think it would be nice to have some examples of some of these fields, or a guide on what type of content goes into the fields. Maybe links to previous evaluations, etc. Could be part of this document or part of the guide doc. I think you've done a great job in the joint evaluation in doing that.